vars/Debian.yml

```diff
@@ -1,6 +1,12 @@
 ---
-__jenkins_repo_url: deb https://pkg.jenkins.io/debian binary/
-__jenkins_repo_key_url: https://pkg.jenkins.io/debian/jenkins.io.key
+__jenkins_repo_url: >-
+  {{ 'deb https://pkg.jenkins.io/debian-stable binary/' if jenkins_prefer_lts | bool else
```

Comment:
Just a style nit - for all these vars, can you make it so the first `{{` occurs on a separate line (so this `'deb ...` should appear on the next line, at the same indentation level as the next `'deb ...` line).

Reply:
I'm the same way.

Reply:
I think I've updated the code to match your preferences. Let me know.